src/options/prop_options.toml

```diff
@@ -64,12 +64,20 @@ name   = "SAT Layer"
   help       = "refine theory conflict clauses (default false)"
 
 [[option]]
-  name       = "minisatUseElim"
+  name       = "minisatVarElim"
```

Comment:
Can we just make all of this a single, 3 mode option?  Having 2 options here seems more confusing.

Reply:
Good point. I'll do that.